3.4/slim/Dockerfile

```diff
@@ -19,7 +19,7 @@ ENV PYTHON_VERSION 3.4.3
 RUN gpg --keyserver pool.sks-keyservers.net --recv-keys 97FC712E4C024BBEA48A61ED3A5CA953F73C700D
 
 RUN set -x \
-	&& buildDeps='curl gcc libc6-dev libsqlite3-dev libssl-dev make xz-utils zlib1g-dev' \
+	&& buildDeps='curl gcc libc6-dev libsqlite3-dev libssl-dev make xz-utils zlib1g-devi libbz2-dev' \
```

Comment:
`devi`? :+1:

(you must be a Vim user! :smile:)

Also, worth noting that these are sorted alphabetically. :wink: